tests/Feature/CelebrityTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+use App\Models\User;
+
+class CelebrityTest extends TestCase
+{
+    /**
+     * A basic feature test example.
+     *
+     * @return void
+     */
+    public function test_example()
+    {
+        $response = $this->get('/');
+
+        $response->assertStatus(200);
+    }
```

Comment:
This test is only testing the index page, so because we are testing the celebrities this test can be deleted. 

By convention, we usually keep only the tests that are relevant to the celebrity here.